src/components/pacbio/PacbioRunWellEdit.vue

```diff
@@ -128,9 +128,9 @@ defineProps({
 
 // Define refs for the component
 // well object
-const well = ref({})
+const well = reactive({ used_aliquots: [], loading_target_p1_plus_p2: '' })
```

Comment:
loading_target_p1_plus_p2 is SMRTLink version specific so it feels like it shouldn't be here.